src/odemis/acq/stream/_sync.py

```diff
@@ -1636,6 +1659,12 @@ def _runAcquisitionScanStage(self, future):
 
             n = 0  # number of points acquired so far
             for px_idx in numpy.ndindex(*rep[::-1]):  # last dim (X) iterates first
+                #TODO K.K. what is both CL and SEM-scanstage are used simoultounously what is displayed? what is called?
```

Comment:
I'm not exactly sure I understand the question, but essentially, the same place is scanned as in the equivalent code in `runAcquisitionEbeam()`. So that code looks fine.